### PR TITLE
cleanup(pubsub): remove handwritten topic admin client from samples.cc

### PR DESCRIPTION
**Pull request by alevenberg**
https://github.com/googleapis/google-cloud-cpp/issues/12987

All of these exist in schema_samples.cc or topic_admin_samples.cc now

This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/13667)

### CODE REVIEW

**Comment by alevenberg** [comment]
Filed https://b.corp.google.com/issues/326958283

**Comment by alevenberg** [comment]
Technically does not run with the emulator, but this was the behavior before and i filed a bug against the emulator

**Review by scotthart — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/13667)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @alevenberg)

___
*[`google/cloud/pubsub/samples/samples.cc` line 39 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/13667#-NrgYVhG-kbFCvlInJy0:-NrgYVhH8sm5qXeD3K6z:b-d6s3um) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/cc57b0cb27716d3df34216bd3766ac4a6b2e9702/google/cloud/pubsub/samples/samples.cc#L39)):*
> ```cpp
> namespace {
> 
> using google::cloud::pubsub::examples::Cleanup;
> ```

Prefer `using` statements to fully qualify symbols, i.e. add the leading `::`.

File: google/cloud/pubsub/samples/samples.cc [hunk 1]
```diff
@@ -37,6 +36,7 @@
 
 namespace {
 
+using google::cloud::pubsub::examples::Cleanup;
```
Comment (alevenberg) [inline google/cloud/pubsub/samples/samples.cc:39]:
```suggestion
using ::google::cloud::pubsub::examples::Cleanup;
```

**Pull request merged**